src/database/schema/ChatSchema.ts

```diff
@@ -172,13 +178,20 @@ class ChatSchema implements BaseDbSchema {
 
   static fromDatabaseObject(dbObject: any): ChatSchema {
     let rawJson = null;
+    let attachmentJson = null;
 
     try {
       rawJson = JSON.parse(dbObject?.raw_json || '{}');
     } catch (e) {
       console.log('error parse');
       console.log(e);
     }
+    try {
+      attachmentJson = JSON.parse(dbObject?.attachment_json || '[]');
+    } catch (e) {
+      console.log('error parse');
+      console.log(e);
+    }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Error handling could be improved here. Currently, if there's an error parsing the JSON, it's logged and execution continues. This might lead to unexpected behavior later on if the parsed data is invalid. Consider throwing an error or handling it in a way that doesn't allow the function to continue with potentially corrupted data.

```diff
-      console.log('error parse');
-      console.log(e);
+      throw new Error('Failed to parse raw_json or attachment_json');
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->